Source/Diffusion.cpp

```diff
@@ -175,13 +176,33 @@ PeleC::getMOLSrcTerm(
         auto const& coe_lambda = coeff_cc.array(dComp_lambda);
         BL_PROFILE("PeleC::get_transport_coeffs()");
         auto const* ltransparm = trans_parms.device_trans_parm();
-        amrex::launch(gbox, [=] AMREX_GPU_DEVICE(amrex::Box const& tbx) {
-          auto trans = pele::physics::PhysicsType::transport();
-          trans.get_transport_coeffs(
-            tbx, qar_yin, qar_Tin, qar_rhoin, coe_rhoD,
-            amrex::Array4<amrex::Real>(), coe_mu, coe_xi, coe_lambda,
-            ltransparm);
-        });
+        auto const& geomdata = geom.data();
+        const ProbParmDevice* lprobparm = PeleC::d_prob_parm_device;
+        const bool get_xi = true, get_mu = true, get_lam = true,
+                   get_Ddiag = true, get_chi = false;
+        amrex::ParallelFor(
+          gbox, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
+            amrex::Real muloc, xiloc, lamloc;
+            amrex::Real Ddiag[NUM_SPECIES], Y[NUM_SPECIES] = {0.0};
+            amrex::Real* chi_mix = nullptr;
+            amrex::Real T = qar_Tin(i, j, k);
+            amrex::Real rho = qar_rhoin(i, j, k);
+            for (int n = 0; n < NUM_SPECIES; ++n) {
+              Y[n] = qar_yin(i, j, k, n);
+            }
+
+            amrex::RealVect x = pc_cmp_loc({AMREX_D_DECL(i, j, k)}, geomdata);
+            pc_transcoeff(
+              get_xi, get_mu, get_lam, get_Ddiag, get_chi, T, rho, Y, Ddiag,
+              chi_mix, muloc, xiloc, lamloc, ltransparm, *lprobparm, x);
```

Comment:
ah I get the trickiness you had to figure out... I guess the only bummer is the extra stuff that needs to be initialized/used here. Duplicate stuff from pelephysics. But there's probably no other good way.